commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPIConfig.java

```diff
@@ -90,6 +91,22 @@ public Impl useLatestNMSVersion(boolean value) {
 		return instance();
 	}
 
+	/**
+	 * Sets whether the CommandAPI should load a (potentially unsupported) NMS version
+	 * when updating to a minor release of Minecraft. As an example, this setting can allow
+	 * updating to 1.21.2 from 1.21.1 but doesn't allow updating to 1.22 from 1.21.2.
+	 * Unlike {@link #useLatestNMSVersion(boolean)}, this setting does
+	 * not blindly load the latest NMS version and as a result breaks cross version compatibility
+	 * but will actually prefer loading the correct NMS implementation.
```

Comment:
This wording might be clearer?

```suggestion
	 * not blindly load the latest NMS version, but will prefer loading the correct NMS implementation when available.
```